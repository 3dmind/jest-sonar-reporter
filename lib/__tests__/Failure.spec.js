'use strict'

const xml = require('xml')
const Failure = require('../Failure')

describe('Failure', () => {
  test('<failure message=""></failure>', () => {
    const mock = 'Lorem ispum'
    const failure = new Failure(mock)

    const actualReport = xml(failure)

    expect(actualReport).toMatchSnapshot()
  })
})
