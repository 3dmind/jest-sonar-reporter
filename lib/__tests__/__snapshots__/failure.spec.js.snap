// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failure <failure message=""></failure> 1`] = `"<failure message=\\"Lorem ispum\\"/>"`;
